rpa-builder/6.2/modules/ROOT/pages/toolbox-network-rest-call.adoc

```diff
@@ -1,170 +1,247 @@
+transferred in the answer.
 
-*ArrayCount* Indicates the number of elements of the returned array.
+* *ArrayCount* 
++
+Indicates the number of elements of the returned array
 
 == Wizard
 
-The wizard is split into the **Settings a**nd *Parameters* areas.
+The wizard is split into the *Settings* and *Parameters* areas.
+
+=== Settings 
+
+image:rest-call-wizard-settings.png[75%, 75%, The Settings section of the REST Call Wizard]
 
 In *Settings*, you specify the *Method Type*, that is, the required HTTP method. In the *Base Url*, you
-specify the Host and Base Path. With the *Url Extension*, you uniquely
-identify the resource.
-You can either just type in the URL or define it via the pin variable.
-//image:media\image2.png[image,width=16,height=20].
+specify the Host and Base Path. With the *Url Extension*, you uniquely identify the resource. You can either just type in the URL or define it via the pin variable. Use *Parse Type* to select between _JsonPath_ and _XPath_ which method to use for parsing the result. The *JsonPath Expression* specifies the path to the property of the response object.
+
+=== Parameters 
 
-//image:media\image3.png[image,width=604,height=25]
+image:rest-call-wizard-parameters.png[75%, 75%, The Parameters section of the REST Call Wizard]
 
 In the *Parameters* area, you provide the required information to the
 end point. There are various options for this, you can enter the
-parameters directly in the URL, similar to the example above, or you can
-transfer them “invisibly” with the different parameters _Request Body_,
-_Header Parameter_ or _Query Parameter_.
+parameters directly in the URL or you can configure the parameters using the different tabs: _Request Body_,
+_Header Parameter_, _Query Parameter_, and _Authentication_.
+
+==== Request Body 
 
 With the *Request Body*, you send data to the server, including, for
 example, the form of communication between the server and the client.
 According to convention, the methods GET, HEAD and DELETE do not require
 a body (see also the network protocol RFC 7231).
 
-In addition, in the Parameters area, on the left of the screen, you can
-find the mapped variables and their *Test Values.* The mapping is
+In the Parameters area you can
+find the mapped variables and their *Test Values*. The mapping is
 carried out via the Pin variable.
-//image:media\image4.png[image,width=15,height=21].
-With the Test Values,
-you can check if the REST Call runs successfully. To do this, having
-entered all the data, click on *_Send_* at the bottom right of the
+
+With the test values,
+you can check if the REST call runs successfully. To do this, having
+entered all the data, click *Send* at the end of the
 wizard.
 
-The (real) values of the mapped variables are not effective until the
+The actual values of the mapped variables are not effective until the
 production run of the Workflow.
 
+===== Inserting Variables Into the Request Body 

```

Comment:
This is the only level 5 heading so I would remove it. It is also deeper than we recommend for heading levels.